src/main/logutils.ts

```diff
@@ -110,48 +165,153 @@ const tailFile = (path: string) => {
     tailHandler.on("line", function(data: string) {
         handleLogLine(data);
     });
-    
+
     tailHandler.on("error", function(error: unknown ) {
       console.log('ERROR: ', error);
     });
-}    
+}
 
 /**
+ * Splits a WoW combat line intelligently with respect to quotes,
+ * lists, tuples, and what have we.
+ *
+ */
+const splitLogLine = (line: string, maxSplits?: number): LogLine => {
```

Comment:
maxSplits should be mentioned in comment I think - not obvious at a glance what it is